.github/workflows/release_workflow.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         with:
           activate-environment: "zppy_publish"
           channel-priority: strict
-          python-version: 3.7
+          python-version: 3.9
```

Comment:
Updated to Python 3.9 (it will pick up the latest patch version, which is 3.9.13).